applications/salk-portal/src/components/AboutCoordAtlasDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import {
+  Box,
+  makeStyles,
+  Grid,
+  Typography,
+  Link,
+  Avatar
+} from "@material-ui/core";
+import { switchActiveColor, headerBorderColor } from "../theme";
+import LOGO from "../assets/images/logo_icon.svg"
+import Modal from "./common/BaseDialog";
+
+const useStyles = makeStyles(() => ({
+    details: {
+      flexGrow: 1,
+      '& .detail-block + .detail-block': {
+        marginTop: '1.5rem',
+      },
+  
+      '& .MuiTypography-root': {
+        fontSize: '0.75rem',
+        lineHeight: '0.9375rem',
+        fontWeight: 600,
+        marginBottom: '0.5rem',
+      },
+
+      '& .MuiLink-root': {
+        color:switchActiveColor,
+        fontWeight: 400,
+      },
+
+      '& p': {
+        '&.MuiTypography-root': {
+          fontWeight: 400,
+        },
+      },
+    },
+
+    avatar: {
+      '& .MuiAvatar-root':{
+        borderColor:headerBorderColor,
+        border: "0.063rem solid",
+        width: "5rem",
+        height: "5rem",
+        borderRadius: "50%",
+        padding:"1rem"
+      }
+      
+    }
+}));
+
+
+
+
+
+const AboutCoordAtlasDialog = (props: any) => {
```

Comment:
@paularah AboutDialog would work too, cause CoordAtlas is already the name of the project